tests/jeos/firstrun.pm

```diff
@@ -44,6 +49,24 @@ sub verify_user_info {
     assert_script_run("ldd --help | grep '^" . $lang_data{$proglang} . "'");
 }
 
+sub verify_mounts {
+    my $expected_type = {mount_type => (expect_mount_by_uuid) ? 'UUID' : 'LABEL'};
+
+    my @findmnt_entries = grep { /\s$expected_type->{mount_type}.*\stranslated\sto\s/ }
+      split(/\n/, script_output('findmnt --verbose --verify'));
+
+    diag "\nfindmnt utility found ( " . scalar(@findmnt_entries) . " ) entries\n";
+    (scalar(@findmnt_entries) > 0) or die "Expected mounts by $expected_type->{mount_type} have not been found\n";
+
+    @findmnt_entries = grep { !/^$expected_type->{mount_type}/ }
+      split(/\n/, script_output('findmnt --fstab --raw --noheadings --df'));
+
+    diag "\nAll mounts should be mounted in the same way\nExceptional mounts found ( " . scalar(@findmnt_entries) . " ) @findmnt_entries\n";
```

Comment:
Seems redundant with the `die` below

Reply:
I have removed both of them, they were useful during development only.